src/front/app/components/Create/Create.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component, PropTypes } from 'react';
+import { reduxForm } from 'redux-form';
+
+const validate  = (values) => {
+    const errors = {};
+
+    if (!values.title) {
+        errors.title = 'Title is required';
+    }
+
+    if(!values.description) {
+        errors.description = 'Description is required';
+    }
+
+    return errors;
+};
+
+const EnabledCheckbox = ({ text }) => (
+    <div className="challenge-create-enabled__title">
+        {text}
+        <div className="challenge-create-enabled__checkbox"><input type="checkbox" checked  enabled/></div>
+    </div>
+);
+
+const DisabledCheckbox = ({ text }) => (
+    <div className="challenge-create-disabled__title">
+        {text}
+        <div className="challenge-create-disabled__checkbox"><input type="checkbox" checked  disabled/></div>
+    </div>
+);
+
+const CreateTitleBlock = ({ text }) => (
+    <span className="challenge-create-titles">{text}</span>
+);
+
+const SetDay = ({ text }) => (
+    <div className="challenge-create-day">
+        <label className="challenge-create-enabled__title">
+            {text}
+            <input className="challenge-create-day-input" type="date" />
+        </label>
+    </div>
+);
+
+
+class CreateForm extends Component  {
```

Comment:
Please do not use shortcuts in component names, fully describe what does this form creates